python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -307,7 +307,7 @@ def _getDefaultValueInternal(self, attr, val=None, check=False):
             from Ganga.GPIDev.Base.Proxy import isType, getRuntimeGPIObject, stripProxy, getName
             from Ganga.GPIDev.Base.Objects import Node
             if isinstance(defvalue, Node):
-                return stripProxy(getRuntimeGPIObject(getName(defvalue)))
+                return allPlugins.find(item['category'], getName(defvalue))()
```

Comment:
The plugin system returns the class object, not any sort of cached object. The `()` at the end are creating a brand new object.